src/utils/string.ts

```diff
@@ -34,3 +35,21 @@ export const convertSymbol = (network: string, value: string) => {
   }
   return value
 }
+
+/**
+ * check url format /network/sym1-to-sym2, sym1 vs sym2 is in whitelist
+ * @param chainId
+ * @param symbol1 ex: knc
+ * @param symbol2 ex: usdt
+ * @returns
+ */
+export const checkPairInWhiteList = (chainId: ChainId | undefined, symbol1: string, symbol2: string) => {
```

Comment:
This func should be moved to `utils/token.ts` or `utils/pair.ts` instead. string is not related here